### ISSUE TITLE
Demo policy links return 404 on GitHub

### ISSUE THREAD

**Issue by ross-spencer** [MediaArea/MediaConch#207]
Looking at the demos page the sample XSL files are returning a 404 error.

E.g. this check, 
https://github.com/MediaArea/MediaConch/tree/master/Demo#general-conformance-with-ffmpeg-generated-matroska-and-ffv1-file

The previous link points to here: https://raw.githubusercontent.com/MediaArea/MediaConch_SourceCode/master/Source/Resource/policies/General_Conformance.xsl

**Comment by ablwr** [comment]
Yeah, this is a known issue. The demo page is essentially "offline" although we didn't delete the URL. How did you come across the page?

**Comment by dericed** [comment]
😬

**Comment by ablwr** [comment]
We are going to be migrating some of this content around soon and cleaning it up  -- sorry for our mess!

**Comment by JeromeMartinez** [comment]
Definitely a need of cleanup :(.

**Comment by ablwr** [comment]
@ross-spencer hit me up directly if you are needing some demo help though!!! happy to do a call or something.

**Comment by ross-spencer** [comment]
@ablwr I'm just exploring the repos to find the publicly available policies - have they there own spot?

**Comment by ablwr** [comment]
All the public policies we know about are here!: https://mediaarea.net/MediaConchOnline/publicPolicies

**Comment by ablwr** [comment]
You can also make your own policy based on an ideal file in the MediaConch GUI based on its MediaInfo report -- if you look at the MediaInfo report and scroll to the bottom, there's a button there to copy policy over from a file. This is often a lot easier than making one from scratch.

**Comment by JeromeMartinez** [comment]
We definitely need to refactor our repos.
For the moment:
- "System" policies (delivered by default) are at https://github.com/MediaArea/MediaConch_SourceCode/tree/master/Tools/Policies
- "Public" policies (from the community) are in a database on the server so not in Git and downloadable at https://mediaarea.net/MediaConchOnline/publicPolicies

**Comment by JeromeMartinez** [comment]
@ablwr "scroll to the bottom" is an outdated advice :-D : no need anymore to scroll to the bottom, same button is available on the top of the windows since last version of MediaConch.

**Comment by ablwr** [comment]
Ahhhh thanks! My tiny UX request. I just did this last weekend when doing a live MediaConch demo so I should have remembered.
